polaris.dme

```diff
@@ -2932,7 +2932,7 @@
 #include "code\ZAS\Zone.dm"
 #include "interface\interface.dm"
 #include "interface\skin.dmf"
-#include "maps\southern_cross\southern_cross.dm"
+#include "maps\example\example.dm"

```

Comment:
Looks like a minor oopsie, assuming this isn't a WIP PR. Surprised Travis didn't fail due to this.

Reply:
Interesting indeed. 
```## UNIT_TEST: --------------- AI Satellite(/area/aisat) lacks an Air scrubber.
## UNIT_TEST: --------------- AI Satellite(/area/aisat) lacks an Air vent.
## UNIT_TEST: !! FAILURE !! [MAP: Area Test APC / Scrubbers / Vents Z level 1]: [1/1]Some areas lacked APCs, Air Scrubbers, or Air vents.
```